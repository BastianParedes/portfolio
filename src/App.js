import 'normalize.css';

import Home from './components/home';
import Nav from './components/nav';
import About from './components/about';
import Portfolio from './components/portfolio';
import Contact from './components/contact';
import Footer from './components/footer';


export default function App() {
    return <>
        <Home />
        <div style={{position: 'relative', backgroundColor: 'rgb(0, 85, 124)'}}>
            <Nav />
            <About />
            <Portfolio />
            <Contact />
        </div>
        <Footer  />
    </>;
}


