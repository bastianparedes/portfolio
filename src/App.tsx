import React from 'react'
import 'normalize.css'
import Home from './components/Home'
import Nav from './components/Nav'
import About from './components/About'
import Portfolio from './components/Portfolio'
import Contact from './components/Contact'
import Footer from './components/Footer'

const App = (): JSX.Element => {
  return (
    <>
      <Home />
      <div
        style={{
          position: 'relative',
          backgroundColor: 'rgb(0, 85, 124)'
        }}
      >
        <Nav />
        <About />
        <Portfolio />
        <Contact />
      </div>
      <Footer />
    </>
  )
}

export default App
