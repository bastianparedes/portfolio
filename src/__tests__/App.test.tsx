import React from 'react'
import App from '../App'
import { render } from '@testing-library/react'

jest.mock('../components/Home', () => () => <div />)
jest.mock('../components/Nav', () => () => <div />)
jest.mock('../components/About', () => () => <div />)
jest.mock('../components/Portfolio', () => () => <div />)
jest.mock('../components/Contact', () => () => <div />)
jest.mock('../components/Footer', () => () => <div />)

describe('<About />', () => {
  it('should render', () => {
    const { container } = render(<App />)
    expect(container).toMatchSnapshot()
  })
})
