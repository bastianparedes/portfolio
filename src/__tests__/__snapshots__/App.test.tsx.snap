// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['<About /> should render 1'] = `
<div>
  <div />
  <div
    style="position: relative; background-color: rgb(0, 85, 124);"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
  <div />
</div>
`;
